### PR TITLE
Backport #132 HL1 frame height workaround

### PR DESCRIPTION
**Pull request by djee-ms**
This backport changes the original code due to some link error on Desktop.
Making a PR for it, then that change will be forward-ported to `master` to fix the same issue there.

### CODE REVIEW

**Comment by djee-ms** [comment]
!azp

File: libs/Microsoft.MixedReality.WebRTC.Native/include/peer_connection.h [hunk 1]
```diff
@@ -244,6 +244,31 @@ class PeerConnection : public webrtc::PeerConnectionObserver,
...
+    /// are added symmetrically at the bottom.
+    PAD = 2
+  };
+
+  /// [HoloLens 1 only]
+  /// Use this function to select whether resolutions where height is not multiple of 16
+  /// should be cropped, padded or left unchanged.
+  /// Default is FrameHeightRoundMode::NONE, except on Hololens 1 where it is
+  /// FrameHeightRoundMode::CROP to avoid severe artifacts produced by
+  /// the H.264 hardware encoder.
```
Comment (fibann) [inline libs/Microsoft.MixedReality.WebRTC.Native/include/peer_connection.h]: Should be changed, now the setting doesn't exist at all on other platforms.

File: libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs [hunk 2]
```diff
@@ -1301,6 +1301,45 @@ public static Task<List<VideoCaptureFormat>> GetVideoCaptureFormatsAsync(string
...
+            /// </summary>
+            PAD = 2
+        }
+
+        /// <summary>
+        /// Use this function to select whether resolutions where height is not multiple of 16
+        /// should be cropped, padded or left unchanged.
+        /// Default is <see cref="FrameHeightRoundMode.NONE"/>, except on Hololens 1 where it is
+        /// <see cref="FrameHeightRoundMode.CROP"/> to avoid severe artifacts produced by
+        /// the H.264 hardware encoder.
```
Comment (fibann) [inline libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs]: Should be same as C++ API.

**Pull request merged**